client/src/components/parts/Header/style.css

```diff
@@ -0,0 +1,46 @@
+.header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 30px;
+    position: fixed;
```

Comment:
I think in mobile don't make header fixed because of the mobile so small.
then, you should make media query only the screen size bigger than 992 have fixed header.
Or you can fix just the hamburger-button (without the site logo).
try to see these sites and use the inspect to understand what I mean :
1- https://socialstreets.co/
2- https://romanroadlondon.com/

Reply:
- :+1:  for the fixed style.
- I did actually changed the hamburger button to become a navbar such as the sites you mentioned but at the last moment I decided to remove it because if we want to edit the page's style to adjust with big screens there will be other styles to apply and I don't think it's a good idea to always load unnecessary styles. (or we could just use the media query for some stuff I don't know). lets talk about it tomorrow/

Reply:
OK